.github/workflows/main.yml

```diff
@@ -63,8 +63,9 @@ jobs:
         shell: bash
         env:
           TAQ_VERSION: "{{ github.ref_name }}"
+          DENO_DIR: "./deno"
         run: |
-          DENO_DIR=./deno deno compile --output taq --target $DENO_TARGET --allow-run --allow-write --allow-read --allow-env index.ts --quickstart "`cat quickstart.md`" --setBuild ${{ github.ref_name }} --setVersion ${{ github.ref_name }}
+          COMMIT=`git rev-parse --short "$GITHUB_SHA"` deno compile --output taq --target $DENO_TARGET --allow-run --allow-write --allow-read --allow-env index.ts --quickstart "`cat quickstart.md`" --setBuild "$TAQ_VERSION/$COMMIT" --setVersion "$TAQ_VERSION"
```

Comment:
Just looking over this I would move the deno compile to the next line to make it slightly more readable as they are two different things

Reply:
Good suggestion. I've never paid all that much attention to this and wasn't aware that I could have multiple commands / one-per-line. Thanks!